CHANGELOG

```diff
@@ -1,3 +1,6 @@
+To be released
+    - Add background-color mixin
+    - Deprecate `base-bg-color`, `bg-color-neutral-30`, `bg-color-error`, and `bg-color-light-error` mixins
```

Comment:
I think we should not be using acronyms for mixin name, `bg`. We should be using `background` for name.